### PR TITLE
Update ut_seattlelib_ntptime.py

### PR DESCRIPTION
**Pull request by karthikgvss**
Changed time interface from time.r2py to time_interface.r2py and added the line for timeport.

### CODE REVIEW

File: tests/ut_seattlelib_ntptime.py [hunk 1]
```diff
@@ -5,15 +5,16 @@
   Function Tested:
     ntp_time_updatetime() in ntp_time.r2py
 """
-    
-from repyportability import *
-add_dy_support(locals())
+#pragma repy restrictions.default dylink.r2py    
+#from repyportability import *
+#add_dy_support(locals())
```
Comment (vladimir-v-diaz) [inline tests/ut_seattlelib_ntptime.py]: Remove lines 9 & 10.

File: tests/ut_seattlelib_ntptime.py [hunk 2]
```diff
@@ -5,15 +5,16 @@
...
 
-time_interface = dy_import_module("time.r2py")
+time_interface = dy_import_module("time_interface.r2py")
 ntp_time = dy_import_module("ntp_time.r2py")
-
+timeport = list(getresources()[0]["connport"])[0]
 try:
-  time_interface.time_updatetime(12345)
+  
+  time_interface.time_updatetime(timeport)
```
Comment (vladimir-v-diaz) [inline tests/ut_seattlelib_ntptime.py]: In the docstring say that ntp_time_updatetime() is indirectly tested / called due to the `dy_import_module("ntp_time.r2py")` statement.

**Comment by karthikgvss** [comment]
Thanks Vlad. Completed the changes. Please check.

**Comment by vladimir-v-diaz** [comment]
Apart from my comment about the exceeded maximum line length, I don't see any other problems with the pull request.  Merging after you fix that one issue.

Test results:
```Bash
$ python utf.py -f ut_seattlelib_ntptime.py 
Testing module: seattlelib
	Running: ut_seattlelib_ntptime.py                           [ PASS ]
$ python
Python 2.7.6 (default, Mar 22 2014, 22:59:56) 
[GCC 4.8.2] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> 
```

**Comment by karthikgvss** [comment]
Made the changes and merged it.

On Wed, Jan 28, 2015 at 2:02 PM, Vladimir Diaz <notifications@github.com>
wrote:

> Apart from my comment about the exceeded maximum line length, I don't see
> any other problems with the pull request. Merging after you fix that one
> issue.
>
> Test results:
>
> $ python utf.py -f ut_seattlelib_ntptime.py
> Testing module: seattlelib
>     Running: ut_seattlelib_ntptime.py                           [ PASS ]
> $ python
> Python 2.7.6 (default, Mar 22 2014, 22:59:56)
> [GCC 4.8.2] on linux2
> Type "help", "copyright", "credits" or "license" for more information.>>>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/SeattleTestbed/seattlelib_v2/pull/158#issuecomment-71894147>
> .
>



-- 


Thanks and regards,
Karthikeya Garimella
Graduate Student - Computer Engineering
Electrical & Computer Engineering Dept.,
New York University - Polytechnic School of Engineering
Mobile - (917) 821-7478

**Pull request merged**